inputs/orchestrator_inner:6.json

```diff
@@ -0,0 +1,204 @@
+{
```

Comment:
These changes break arrangement 5 which is fine. But it makes adding a new arrangement not needed.

Reply:
As long as we say in the release notes for osbs-client that it will require a newer version of atomic-reactor (i.e. one in wihch the plugin won't expect those parameters), then it's fine to stick with the existing plugin arrangement.

In any case, adding a new arrangement doesn't necessarily help the plugins to know whether to expect the parameters in question, as they don't necessarily know the arrangement version they are running for.

Reply:
ok removing 6, and modifying 5